SWI-cpp2.h

```diff
@@ -1747,15 +1748,15 @@ class PlBlobV
   static void acquire(atom_t a)
   { PlAtom a_(a);
     auto data = cast_check(a_);
-    bool rc;
+    bool rc = false; // Uninitialized variable warning (some compilers)
```

Comment:
There are a number of these changes, all essentially the same, but only one triggered a warning from gcc-11.
I decided to make the same change to all for consistency, and also because I think it makes the code slightly clearer (if tracing through C++ try/catch can be considered "clear").